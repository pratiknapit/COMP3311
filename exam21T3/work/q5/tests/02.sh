python3 ./q5 House 5000000 5 4 3
