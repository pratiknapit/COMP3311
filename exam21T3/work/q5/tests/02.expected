#46358: 5 Aboud Avenue, Kingsford 2032, 5br, 4ba, 3car, $2296000
#46520: 69 Bay Street, Coogee 2034, 5br, 4ba, 3car, $2704000
